web/src/engine/websites/Qimeitj.ts

```diff
@@ -0,0 +1,41 @@
+import { Tags } from '../Tags';
+import icon from './Qimeitj.webp';
+import { type Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as MH from './decorators/MH';
+import { FetchCSS } from '../platform/FetchProvider';
+
+@Common.MangaCSS(/^{origin}\/book\/[^/]+$/, MH.queryMangaTitleFromURI)
+@Common.MangasMultiPageCSS(MH.mangaPath, MH.queryMangas)
+@Common.ChaptersSinglePageCSS(MH.queryChapters, MH.ChapterExtractor)
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('qimeitj', `Qimeitj`, 'https://qimeitj.com', Tags.Language.Chinese, Tags.Media.Manhua, Tags.Media.Manhwa, Tags.Source.Aggregator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchPages(chapter: Chapter): Promise<Page[]> {
+        const pagesList: Page[] = [];
+        for (let page = 1, run = true; run; page++) {
+            const pages = await this.getPagesFromChapterPage(page, chapter);
+            pages.length > 0 ? pagesList.push(...pages) : run = false;
+        }
+        return pagesList;
+    }
+    async getPagesFromChapterPage(page: number, chapter: Chapter): Promise<Page[]> {
+        const url = new URL(chapter.Identifier, this.URI);
+        url.search = new URLSearchParams({
```

Comment:
Since this is just a single parameter `url.searchParam.set()` seems to be more appropriate